rest-service/manager_rest/configure_manager.py

```diff
@@ -306,8 +306,7 @@ def _register_rabbitmq_brokers(user_config):
 
         # reload config after inserting rabbitmqs, so that .amqp_host
         # and others are set
-        db.session.commit()
```

Comment:
Why don't you need to commit any more here?

Reply:
We used to need to commit, because `.load_from_db` created a new connection to the db, and so it of course didn't see uncommitted entries.
Now, we pass through the session, so `.load_from_db` runs its queries in the context of the currently-open transaction anyway, so it does see the rows that aren't committed yet.